qiskit_optimization/translators/ising.py

```diff
@@ -32,6 +36,8 @@ def to_ising(quad_prog: QuadraticProgram) -> Tuple[OperatorBase, float]:
 
     Args:
         quad_prog: The problem to be translated.
+        opflow: The output object is an OpFlow's operator if True.
```

Comment:
Juts checking on `is an OpFlow's operator`  - the output here is always a PauliSumOp right, may be we can say then `an opflow PauliSumOp`. In practice I think it has to be a PauliSumOp since thats the only opflow operator listed by primitives as supported i.e. `BaseOperator | PauliSumOp`

Reply:
I updated the code so that the return object can be always PauliSumOp with `opflow=True`.